crates/wasmparser/src/readers/core/types.rs

```diff
@@ -1820,6 +1867,11 @@ impl<'a> FromReader<'a> for HeapType {
                     let ty = reader.read()?;
                     Ok(HeapType::Abstract { shared: true, ty })
                 }
+                0x62 => {
```

Comment:
Mind updating the table [here](https://github.com/bytecodealliance/wasm-tools/blob/07068fa17507e38310bcbefa83a001423c7548d9/crates/wasmparser/src/readers/core/types.rs#L1725-L1775) as well?